Description: Drop http3 test as Debian doesn't build curl with http3 support
Forwarded: not-needed
Author: Samuel Henrique <samueloph@debian.org>
Index: pycurl/tests/option_constants_test.py
===================================================================
--- pycurl.orig/tests/option_constants_test.py
+++ pycurl/tests/option_constants_test.py
@@ -496,6 +496,7 @@ class OptionConstantsSettingTest(unittes
     def test_http_version_2prior_knowledge(self):
         self.curl.setopt(self.curl.HTTP_VERSION, self.curl.CURL_HTTP_VERSION_2_PRIOR_KNOWLEDGE)
 
+    @pytest.mark.skip(reason="Debian doesn't build curl with http3 support (it's experimental)")
     @util.min_libcurl(7, 66, 0)
     def test_http_version_3(self):
         self.curl.setopt(self.curl.HTTP_VERSION, self.curl.CURL_HTTP_VERSION_3)
